### PR TITLE
[DS-142]: Feat/ds 142 external link

### PR DESCRIPTION
**Pull request by michalisKout**
## Description
- adds new external link in asset heading
- adds new lib (tippyjs) for handling common tooltip cases
[jira task](https://orfium.atlassian.net/browse/DS-142)

## Screenshot

ezgif com-gif-maker (2)

## Test Plan

- snapshot update

### CODE REVIEW

**Comment by michalisKout** [comment]
@panvourtsis  added a todo based on the new [story for the tooltip](https://orfium.atlassian.net/browse/DS-153)

File: src/components/AssetMatching/components/Asset/Asset.style.ts [hunk 1]
```diff
@@ -73,7 +74,45 @@ const linkedCount = (theme: Theme): SerializedStyles => css`
...
+        > path {
+          ${transition(0.2, 'fill')};
+          fill: ${theme.palette.black};
+        }
+      }
+    }
+  }
+`;
+
+export const title = css`
```
Comment (panvourtsis) [inline src/components/AssetMatching/components/Asset/Asset.style.ts:95]: i am confused export here and by default ?

File: src/components/AssetMatching/components/Asset/AssetHeading.tsx [hunk 2]
```diff
@@ -3,20 +3,32 @@ import Styles from './Asset.style';
...
+  externalLink?: ExternalLinkProps;
   bottom?: string;
   iconName: AcceptedIconNames;
 }
-const AssetHeading: FC<Props> = ({ top = '', main, bottom = '', iconName }) => {
+
+const AssetHeading: FC<AssetHeadingProps> = ({
+  top = '',
+  main,
+  bottom = '',
```
Comment (panvourtsis) [inline src/components/AssetMatching/components/Asset/AssetHeading.tsx:19]: i dont get why we have these empty strings here? you dont use it

Comment (michalisKout) [reply src/components/AssetMatching/components/Asset/AssetHeading.tsx:19]: Just passing by default empty strings, we can enhance these props with pre-defined text in the future.

**Review by panvourtsis — APPROVED** [review]

**Pull request merged**